includes/WikiPage.php

```diff
@@ -590,9 +590,11 @@ public function isCountable( $editInfo = false ) {
 	 *
 	 * The target will be fetched from the redirect table if possible.
 	 * If this page doesn't have an entry there, call insertRedirect()
+	 *
+	 * @param bool $userMaster
 	 * @return Title|mixed object, or null if this page is not a redirect
 	 */
-	public function getRedirectTarget() {
+	public function getRedirectTarget( $userMaster = false ) {
```

Comment:
I know its pretty unintuitive, but we should probably use ```\Title::GAID_FOR_UPDATE``` for this kind of stuff like the rest of this class and many other core classes do.